gsadjust/data_analysis.py

```diff
@@ -245,27 +227,31 @@ def adjusted_vs_observed_datum_analysis(self):
             done = False
             adj_g = None
             for ii in range(obstreesurvey.datum_model.rowCount()):
-                if not done:
```

Comment:
As before, avoid nesting the default path. The test for `done` is moved below, and triggers a `break` out of the outer loop.